docs/design/operator.md

```diff
@@ -98,7 +98,7 @@ spec:
     enableMetadata: true
     grpcTimeout: 100
     snapshotPolicy: auto-detect
-    generateOMapInfo: true
+    generateOMapInfo: false
```

Comment:
This is a user-facing option to enable the feature, let's revert this change.

Reply:
done